pkgs/development/compilers/swift/wrapper/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, swift
+, wrapperParams ? swift._wrapperParams
+, useSwiftDriver ? true, swift-driver
```

Comment:
I think this is only useful if `swift-driver` is nullable, which in turn would only be useful if it was something a user has to explicitly inject in `callPackage`. In this case, `swift-driver` is always present, because it is already part of the swift scope.